examples/cpp/executor/CMakeLists.txt

```diff
@@ -13,11 +13,13 @@
 # License for the specific language governing permissions and limitations under
 # the License. cmake needs this line
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.27)
 
 set(TRTLLM_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../..")
-include(${TRTLLM_DIR}/cpp/cmake/modules/set_ifndef.cmake)
-include(${TRTLLM_DIR}/cpp/cmake/modules/find_library_create_target.cmake)
+list(APPEND CMAKE_MODULE_PATH "${TRTLLM_DIR}/cpp/cmake/modules")
+
+include(set_ifndef)
```

Comment:
I don't think `set_ifndef` is used in this file.

Reply:
Nice catch. I realized that we no longer using this macro anywhere, so we can get rid of it.